design/common.blocks/link/_theme/link_theme_islands.styl

```diff
@@ -18,6 +18,21 @@
         {
             color: #070;
         }
+
+        &_ghost
+        {
+            color: #999;
+        }
+
+        &_black
+        {
+            color: #000;
+        }
+
+        &_strong
```

Comment:
It's a way to keep everything consistent (see posts about different colors in header at some small search company ;))